doc/list.rst

```diff
@@ -2088,7 +2088,19 @@ List of Available Rules
    `Source PhpCsFixer\\Fixer\\Phpdoc\\PhpdocScalarFixer <./../src/Fixer/Phpdoc/PhpdocScalarFixer.php>`_
 -  `phpdoc_separation <./rules/phpdoc/phpdoc_separation.rst>`_
 
-   Annotations in PHPDoc should be grouped together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line.
+   Annotations in PHPDoc should be grouped together so that annotations of the same type immediately follow each other. Annotations of a different type are separated by a single blank line.
+
+   Configuration options:
+
+   - | ``groups``
+     | Sets of annotation types to be grouped together.
+     | Allowed types: ``string[][]``
+     | Default value: ``[['deprecated', 'link', 'see', 'since'], ['author', 'copyright', 'license'], ['category', 'package', 'subpackage'], ['property', 'property-read', 'property-write']]``
+   - | ``psr_standard_tags_only``
+     | Whether to only process annotations defined by PSR-5 draft, which are: `api`, `author`, `category`, `copyright`, `deprecated`, `example`, `global`, `internal`, `license`, `link`, `method`, `package`, `param`, `property`, `property-read`, `property-write`, `return`, `see`, `since`, `subpackage`, `throws`, `todo`, `uses`, `var`, `version`.
+     | Allowed types: ``bool``
+     | Default value: ``true``
```

Comment:
1) There is something unclear about this config

What if I add `@phpstan-param` in the `groups` option, but I didn't changed psr_standard_tags_only to true.

Will it be ignored or fixed ?

2) I think something like a `supported_tags` would be better than the `psr_standard_tags_only` ;

What if I want to run this rule on only some of the psr tags ?
What if I want to run this rule on all the psr tags and only some extra ones ?

Wouldn't be better to configure it this way
```
'supported_tags' => [
    'param', 'return', 'phpstan-param', 'phpstan-return', 'throws',
]
```

Reply:
The `psr_standard_tags_only` option might be dropped, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6501#discussion_r945132658

Reply:
@julienfalque option dropped.